### PR TITLE
feat(crons): Basic scafolding for overviewTimeline

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/monitors/components/overviewTimeline/index.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      ))}
+    </MonitorListPanel>
+  );
+}
+
+function MonitorDetails({monitor}: {monitor: Monitor}) {
+  const schedule = scheduleAsText(monitor.config);
+
+  return (
+    <DetailsContainer to="/crons/">
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/index.tsx:65]: should this go to each cron details page?

File: static/app/views/monitors/components/overviewTimeline/index.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  display: grid;
+  grid-template-columns: 350px 1fr;
+`;
+
+const TimelineContainer = styled('div')`
+  border-bottom: 1px solid ${p => p.theme.innerBorder};
+`;
+
+const DetailsContainer = styled(Link)`
+  color: ${p => p.theme.textColor};
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/index.tsx:82]: For some reason I see some border-radius on this element, can we override? Not sure where this is coming from though :/

image

File: static/app/views/monitors/components/overviewTimeline/index.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  padding: ${space(2)};
+  border-right: 1px solid ${p => p.theme.border};
+  border-bottom: 1px solid ${p => p.theme.innerBorder};
```
Comment (davidenwang) [inline static/app/views/monitors/components/overviewTimeline/index.tsx:85]: Mocks dont seem to have border bottom here, are we including it like this though to help separate the listings?

**Review by davidenwang — APPROVED** [review]

File: static/app/views/monitors/components/overviewTimeline/index.tsx (hunk 3 above)
Comment (evanpurkhiser) [reply static/app/views/monitors/components/overviewTimeline/index.tsx]: I was just playing with it, feel free to remove it

File: static/app/views/monitors/components/overviewTimeline/index.tsx [hunk 4]
```diff
@@ -59,10 +60,13 @@ export function OverviewTimeline({monitorList}: Props) {
 }
 
 function MonitorDetails({monitor}: {monitor: Monitor}) {
+  const organization = useOrganization();
   const schedule = scheduleAsText(monitor.config);
 
+  const monitorDetailUrl = `/organizations/${organization.slug}/crons/${monitor.slug}/`;
```
Comment (evanpurkhiser) [inline static/app/views/monitors/components/overviewTimeline/index.tsx:66]: I think we need `normalizeUrl` here

**Pull request merged**